scripts/build-agent-sea-win64.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Fail on error
+set -e
+
+# Get the current version number
+export MEDPLUM_VERSION=$(node -p "require('./package.json').version")
+
+# Move into packages/agent
+pushd packages/agent
+
+# Build the agent
+npm run build
+
+# # Check Node version
+# # If not the expected default version, then warn
+# nodeversion=$(node -v)
+# if [$(echo nodeversion | cut -d '.' -f 1) != "v20"]; then
+#   echo "Expected to be on v20.x.x but on $(node -v)"
+# fi
+
+# Generate blob to inject into node executable
+node --experimental-sea-config sea-config.json
+
+# Copy the local node binary
+node -e "require('fs').copyFileSync(process.execPath, 'dist/medplum-agent-$MEDPLUM_VERSION-win64.exe')" 
+
+# Remove signature from binary
+signtool remove -s dist/medplum-agent-$MEDPLUM_VERSION-win64.exe
```

Comment:
Add comment about how to install `signtool` from Windows SDK, if missing:

References: 
https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/
https://stackoverflow.com/a/52963704